src/tactic/spass/examples.lean

```diff
@@ -0,0 +1,115 @@
+/-
+  Copyright (c) 2019 Seul Baek. All rights reserved.
+  Released under Apache 2.0 license as described in the file LICENSE.
+  Author: Seul Baek
+-/
+
+import tactic.spass.main
+
+section
+
+/- Examples from finish. -/
+
+variables {A B C D : Prop}
+
+  example : ¬ A → A → C := by spass
+  example : (A ∧ A ∧ B) → (A ∧ C ∧ B) → A := by spass
+  example : A → ¬ B → ¬ (A → B) := by spass
```

Comment:
After updating to spass 3.9, most tests pass (a few timeout), but this one fails with error message:
```
failed
state:
A B : Prop
⊢ ¬A ∨ B ∨ A ∧ ¬B
```

Reply:
Huh, that's strange. Can't replicate the behavior on my side.

Reply:
Sometimes `spass` doesn't succeed on the first try even when a proof is found (I think this might have to do with the i/o not getting properly updated) but it works if you run it again. Perhaps this is a similar case?